scripts/script.sh

```diff
@@ -0,0 +1,4 @@
+sleep 4
+mix ecto.migrate
+python3 /fixtures/main.py -path /fixtures/jpks/
```

Comment:
I guess we want to populate database only once. We don't want to run this script whenever app is starting.